db/migrate/20200226210111_validate_create_email_authorization_table.rb

```diff
@@ -0,0 +1,5 @@
+class ValidateCreateEmailAuthorizationTable < ActiveRecord::Migration[5.2]
+  def change
+    validate_foreign_key :email_authorizations, :users
```

Comment:
is this needed for a newly created table?

Reply:
Yeah, I got thrown an error message when I tried to do a "regular" foreign key addition. See https://github.com/ankane/strong_migrations#adding-a-foreign-key

Reply:
TIL, thanks!